docker/runner.Dockerfile

```
@@ -0,0 +1,20 @@
+FROM ghcr.io/actions/actions-runner:2.316.1
```

Switching to Ubuntu 20.04 seems to be as easy as:

```suggestion
FROM ghcr.io/actions/actions-runner:v2.316.1-ubuntu-20.04
```